### ISSUE TITLE
UI: Initial UI dev appears to be broken

### ISSUE THREAD

**Issue by dselans** [9corp/9volt#83]
When performing `make run/ui` for the first time - the ui will fail to load until you perform a `make build` first.

This is likely something silly and (hopefully) simple.

@caledhwa mind taking this?

**Comment by caledhwa** [comment]
Yep, I'll take this. I may have to find a empty env to work in though :( - Definitely worth it though.

**Comment by dselans** [comment]
Assigning it to you - more visible/obvious in the board.
